packages/css-data/bin/mdn-data.ts

```diff
@@ -57,12 +37,62 @@ const normalizedValues = {
     value: 0,
     unit: "px",
   },
-  // https://github.com/mdn/data/issues/556
-  // @todo remove once fixed
   "background-size": autoValue,
   "text-size-adjust": autoValue,
 } as const;
 
+const parseInitialValue = (property: string, value: string): StyleValue => {
```

Comment:
this script starts to become more and more complex, I think its time to take out functions and add tests for them

Reply:
__generated__ here is the result of this script. Kinda snapshot test already.

Reply:
its a very complex snapshot, once it changes, you won't have any clue why :) and its easy to overlook